templates/zerver/for-education.html

```diff
@@ -0,0 +1,254 @@
+{% extends "zerver/portico.html" %}
+{% set entrypoint = "landing-page" %}
+
+{% set OPEN_GRAPH_TITLE = 'Teach a course with Zulip.' %}
+{% set OPEN_GRAPH_DESCRIPTION = 'Online, in-person, and anything in between' %}
```

Comment:
We can do these as follow-ups, but we should probably put all of these pages into the Facebook/Twitter tools and make sure we like the resulting look.

Reply:
Ok.